ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywallView.kt

```diff
@@ -47,15 +44,6 @@ internal fun InternalPaywallView(
     }
 }
 
-@Composable
-private fun updateStateIfLocaleChanged(viewModel: PaywallViewModel) {
-    var locale by remember { mutableStateOf(LocaleListCompat.getDefault()) }
```

Comment:
As mentioned in the description, this doesn't survive configuration changes. So we were always getting the latest locales upon a locale change.